File: src/codes/Users_Manual/link/visualizer_mod.py
~from pyautd3 import AUTD3, Controller, Segment, Hz
~from pyautd3.link.visualizer import PlotConfig, Visualizer
~from pyautd3.modulation import Sine
autd: Controller[Visualizer] = (
    Controller.builder([AUTD3([0.0, 0.0, 0.0])]).open(Visualizer.builder())
)

m = Sine(150 * Hz)
autd.send(m)

autd.link.plot_modulation(PlotConfig(fname="mod.png"), Segment.S0)

File: src/codes/Users_Manual/link/visualizer_mod.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Gain;
~using AUTD3Sharp.Modulation;
~using AUTD3Sharp.Utils;
~using static AUTD3Sharp.Units;
var autd = new ControllerBuilder().AddDevice(new AUTD3(Vector3d.Zero)).Open(Visualizer.Builder());

var m = new Sine(150u * Hz);
autd.Send(m);

autd.Link.PlotModulation(new PlotConfig
{
    Fname = "mod.png"
}, Segment.S0);